sdks/node-sdk/src/Client.ts

```diff
@@ -292,6 +292,35 @@ export class Client {
     }
   }
 
+  /**
+   * WARNING: This function should be used with caution. It is only provided
+   * for use in special cases where the provided workflows do not meet the
+   * requirements of an application.
+   *
+   * It is highly recommended to use the `changeRecoveryIdentifer` function instead.
```

Comment:
Followed the pattern with this comment here, but seems like the other recommended (non unsafe) functions in this file don't exist either. Would now be a good time to update these comments, or do these functions exist  somewhere that I am missing? @rygine